lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -122,21 +123,21 @@ public void onResult(Throwable result) {
 
   public LottieAnimationView(Context context) {
     super(context);
-    init(null);
+    init(null, R.attr.lottieAnimationViewStyle);
```

Comment:
This matches what I think is the convention - to specify the default style attr even if programmatically creating the view.

I think it's conventional (at least in MaterialDesignComponents-Android/AppCompat) to chain constructors:

```java
public LottieAnimationView(Context context) {
  this(context, null);
}

public LottieAnimationView(Context context, AttributeSet attrs) {
  this(context, attrs, R.attr.lottieAnimationViewStyle);
}

public LottieAnimationView(Context context, AttributeSet attrs, int defStyleAttr) {
  super(context, attrs, defStyleAttr);
  init(attrs, defStyleAttr);
}
```

but I left it as-is, since I'm not sure what the consequences are if someone was relying on the fact that they don't chain.